docs/commands/convert.md

```diff
@@ -0,0 +1,12 @@
+# Convert existing oscal documents to another version
```

Comment:
Worth Consideration:

OSCAL-CLI uses `convert` to convert between different file types (yaml, json, xml). I see us having support for JSON natively in Lula/go-oscal.

Do we ever intend to need support at the go-oscal layer for this conversion?

Reply:
I think changing the command and verbiage to upgrade would be a better. I was using the command that was there and though maybe that choice had already been made but I am glad to pick an alternative, like upgrade or update or if you have any ideas.